### DISCUSSION TITLE
Is an update to the "Mandatory Tutorial" possible

### DISCUSSION THREAD

**Discussion by Menelioa** [UltimaHoarder/UltimaScraper#1071] [Q&A] [answered]
Hi, I've been using this data scrapper, for along time (thanks by the way), but I haven't updated my version for awhile cuz the last time I used it everything worked fine. However when I tried to use it recently it got an Auth error. I decide that getting the latest version would be the solution, but it seems like a lot has changed and I'm not sure how to begin to set stuff up. For example I can't seem to find the auth.json anywhere. Would help a lot if the "Mandatory Tutorial" was updated.

**Comment by secretiveolwagner** [comment, marked as answer]
[It is updated my dude](https://github.com/DIGITALCRIMINAL/OnlyFans#mandatory-tutorial)

@DIGITALCRIMINAL please mark as answered thank you.
